hera_pspec/tests/test_pspecbeam.py

```diff
@@ -99,14 +102,12 @@ def test_Gaussbeam(self):
         self.assertAlmostEqual(Om_pp[4], 0.36258881134617554)
         self.assertAlmostEqual(Om_pp[4], Om_pp[0])
 
-        self.assertAlmostEqual(scalar/10**5, 6392750121.05/10**5)
+        self.assertAlmostEqual(scalar/6082814757.7556648, 1.0, delta=1e-4)
         
         # convergence of integral
-        scalar_double_steps = self.gauss.compute_pspec_scalar(lower_freq, 
-                                        upper_freq, num_freqs, num_steps=20000) 
-        # assertAlmostEqual does absolute comparisons, so we want to put things 
-        # on a sensible scale
-        scalar /= 10**9
-        scalar_double_steps /= 10**9
-        self.assertAlmostEqual(scalar, scalar_double_steps)
+        scalar_large_Nsteps = self.gauss.compute_pspec_scalar(lower_freq, upper_freq, num_freqs, num_steps=5000)
+        self.assertAlmostEqual(scalar / scalar_large_Nsteps, 1.0, delta=1e-5)
+
+        # test taper execution
+        scalar = self.bm.compute_pspec_scalar(lower_freq, upper_freq, num_freqs, num_steps=2000, taper='blackman')
```

Comment:
Can we add a numerical test for this as well? Just to make sure that when the taper is applied the number is correct. I'm happy to be the one to add this if it's easier for you; just let me know!

Reply:
do we have an analytic number to compare to? if you do feel free to add it. otherwise I'm just going to see what that number is (given the current format of `_compute_pspec_scalar` and the input args) and put in an `almost_equal` float comparison, which I think is how we're currently doing it...

Reply:
either way feel free to do it yourself if you'd like!

Reply:
did it